### ISSUE TITLE
/SDF/GET_GENERIC_APO_TABLE does not work (caused by SAP change)

### ISSUE THREAD

**Issue by icoxfog417** [icoxfog417/SAPExtractorDotNET#4]
I don't know what version from , but at least SAP_AP 700 SP29 , `/SDF/GET_GENERIC_APO_TABLE` doesn't work because of it's modification.

Please see below, EXIT function before process execute...

``` abap
 DATA: no_lines TYPE i.
 DATA: NR_COLUMNS TYPE I,
       FDPOS          LIKE SY-FDPOS,
       LEN            TYPE I,
       CONENV         LIKE /SDF/LC_DBCON-CON_ENV.
 DATA:
        where_condition(1000) TYPE  c,
        select_columns LIKE where_condition,
        wclause LIKE where_condition.

    REFRESH int_tab .


    EXIT.

*---------- order the select and where condition ------------
    wclause = ''.
```

`RFC_READ_TABLE` is still work , so have to change function to it.

**Comment by keitaro-akiyama** [comment]
can use the "SAP Query" in the new SAP ?

**Comment by icoxfog417** [comment]
I don't confirm it, but the remote function in SAP is active.
I'll check it until the end of month (maybe).

**Comment by icoxfog417** [comment]
fix at 0da5a4ed84bda1a812bc28c90370eca6344731f1
(made mistake on issue no... not #1, but #4)
